self-api/src/main/java/com/selfxdsd/api/Projects.java

```diff
@@ -69,4 +71,12 @@ Project getProjectById(
         final String repoFullName,
         final String repoProvider
     );
+
+    /**
+     * Get the Projects at the provided Page.
+     * @param page Page number.
+     * @return ProjectsPaged
+     */
+    ProjectsPaged page(final Paged.Page page);
```

Comment:
@criske I think ``Projects page(final Paged.Page page)`` would make more sense :D

Reply:
@amihaiemil in this draft `ProjectsPaged` is needed because the frontend needs the Paged "header" in order to construct the ui paging links from 1 to totalpages and highlight the current page.